returnn/tf/layers/signal_processing.py

```diff
@@ -426,32 +477,52 @@ def _get_nr_of_channels_from_input_placeholder(self):
 
   def _apply_stft_to_input(self):
     from returnn.tf.util.basic import get_shape
-    def _cropStftOutputToReferenceFrameSizeLength(channel_concatenated_stft, crop_size):
-      return tf.slice(channel_concatenated_stft, [0, 0, 0], [get_shape(channel_concatenated_stft)[0], crop_size, get_shape(channel_concatenated_stft)[2]])
+
+    # noinspection PyShadowingNames
+    def _crop_stft_output_to_reference_frame_size_length(channel_concatenated_stft, crop_size):
+      return tf.slice(channel_concatenated_stft, [0, 0, 0], [get_shape(channel_concatenated_stft)[0], crop_size,
+                                                             get_shape(channel_concatenated_stft)[2]])
```

Comment:
You should never indent like that. Rather, add newline after `tf.slice(` and then indent properly.